src/index.js

```diff
@@ -67,6 +67,53 @@ VirtualMachine.prototype.greenFlag = function () {
 };
 
 /**
+ * Set whether the VM is in "turbo mode."
+ * When true, loops don't yield to redraw.
+ * @param {Boolean} turboModeOn Whether turbo mode should be set.
+ */
+VirtualMachine.prototype.setTurboMode = function (turboModeOn) {
+    this.runtime.turboMode = (turboModeOn == true);
```

Comment:
Just curious: why `(turboModeOn == true)` rather than, say, `!!turboModeOn`? Not that it should matter :)